package-metadata.json

```diff
@@ -20,6 +20,8 @@
     "SUPERSET_SSL": "false",
 
     "FHIR_IG_URL": "https://openhie.github.io/cares-covid-ig",
```

Comment:
interoperability-layer-openhim should be in the dependencies

Reply:
Thanks, I agree with you but this is not related to this pr. You would also need to add a config importer for creating the openhim channel that will be used. That work should be done under a different ticket